src/common/normalize.css

```diff
@@ -4,12 +4,18 @@
 
 body {
   overflow-x: hidden;
-}
 
-html,
-body {
   color: #403f4c;
 
   font-family: colfax-web, sans-serif;
+  font-size: 16px;
```

Comment:
Are these values on variables somewhere? Just a question

Reply:
Nope, this project does not use any variables whatsoever, both for flexibility and speed at the time this design was implemented.

Reply:
Yes 👍